src/include/evict_inline.h

```diff
@@ -0,0 +1,415 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#pragma once
+
+/*
+ * __wt_cache_aggressive --
+ *     Indicate if the cache is operating in aggressive mode.
+ */
+static WT_INLINE bool
+__wt_cache_aggressive(WT_SESSION_IMPL *session)
```

Comment:
Are we planning to rename these functions in a later PR? I don't think the `__wt_cache` prefix makes sense any more.

The one exception to this is `__wt_cache_stuck` since cache stuck is common terminology. Maybe that could become `__wt_evict_cache_stuck`

Reply:
That's correct @ajmorton, the function names will be one PR, where we decide on the public, private APIs, removing the extra stuff. Then we can get to naming (this should happen soon). 

For now, it is simply splitting the functions.